Source/WebKit/Platform/IPC/SharedBufferReference.cpp

```diff
@@ -48,12 +48,13 @@ void SharedBufferReference::encode(Encoder& encoder) const
     if (!m_size)
         return;
 
-    SharedMemory::Handle handle;
-    {
-        auto sharedMemoryBuffer = m_memory ? m_memory : SharedMemory::copyBuffer(*m_buffer);
-        sharedMemoryBuffer->createHandle(handle, SharedMemory::Protection::ReadOnly);
+    auto sharedMemoryBuffer = m_memory ? m_memory : SharedMemory::copyBuffer(*m_buffer);
```

Comment:
this loses the (although future) that the shmem refcount be one by the time we send it over IPC. 
This will be required once xnu implements the ability to transfer a shmem from one process to another (and so it becomes read-only in the sender process)

Reply:
So please either link to a follow-up bug and add a comment about it, or keep that requirement.